### ISSUE TITLE
Automate invoicing of new corporate members

### ISSUE THREAD

**Issue by timgraham** [django/djangoproject.com#602]
It should be feasible to automatically invoice new corporate members when they sign up. Assigning to Adrienne to provide a sample invoice. Should we always invoicing in the $2,000/$25,0000/$100,000 amounts or should we allow the member to input how much they wish to contribute?

**Comment by adriennefriend** [comment]
Hi Tim, thanks for your work on this! I'm not sure if I will be the one who will invoice corporate members. That has typically been the work of the DSF Treasurer, and I'm not sure if that will continue once the new board is in place. I agree that automating the process would be helpful, but right now I don't have a sample invoice to provide, and don't want to make decisions on someone else's behalf (if this work is managed by a DSF Board member).

**Comment by timgraham** [comment]
Sounds good, no rush on this from my perspective. We'll work with the new treasurer to figure out a good system.

**Comment by adriennefriend** [comment]
Thank you so much, Tim! I'm mentioning this Issue in my report to the board tonight, letting them know that this is something you're thinking about/have done some work on.

**Comment by adriennefriend** [comment]
The form should note the requirements for membership (at least 2K for Silver, at least 25K for Gold, at least 100k for Platinum), but we should allow them to input what they wish to contribute, not restrict them to those amounts. If someone wants to give more, it should be an option.

**Comment by adriennefriend** [comment]
Still discussing with Board. For now, we're doing it old style. Once the new Treasurer is oriented, we should resume discussions. Perhaps February.

**Comment by timgraham** [comment]
New treasurer Christophe indicated he would like to do this manually, so closing as wontfix for now.
